.github/workflows/add_issue_to_project.yml

```
@@ -6,12 +6,14 @@ on:
   issues:
     types:
       - opened
-      - transfered
+      - transferred
 
 jobs:
   add-to-project:
     name: Add issue to project Rancher/Fleet
     runs-on: ubuntu-latest
+    if: >
+      github.repository == 'rancher/fleet'
```

```suggestion
      github.repository == 'rancher/gitjob'
```

I think it's the source repo.